SCAutolib/models/card.py

```diff
@@ -91,6 +91,8 @@ def __init__(self, user: User, insert: bool = False,
         self.user = user
         assert self.user.card_dir.exists(), "Card root directory doesn't exists"
 
+        self.dump_file = LIB_DUMP_CARDS.joinpath(f"card-{user.username}.json")
```

Comment:
Support for dumping is added, but not for loading. Is that intentional?

Reply:
No, there is support for `load` functionality. Check [here](https://github.com/x00Pavel/SCAutolib/blob/db4a3c1095dc8055baff05db15c768e2a230ad29/SCAutolib/models/user.py#L37)

Reply:
Oh, sorry. Then yes, this is intentional. This functionality is implemented in another [PR](https://github.com/x00Pavel/SCAutolib/pull/87)